src/toil/job.py

```diff
@@ -1402,41 +2351,61 @@ def _executor(self, jobGraph, stats, fileStore):
                 )
             )
 
-    def _runner(self, jobGraph, jobStore, fileStore, defer):
+    def _runner(self, jobStore=None, fileStore=None, defer=None, **kwargs):
         """
         This method actually runs the job, and serialises the next jobs.
+        
+        It marks the job as completed (by clearing its command) and creates the
```

Comment:
Thanks again for the clear, well-explained docstring additions.

Reply:
I write these because I'm totally lost otherwise.